www/src/layouts/docs.astro

```diff
@@ -39,6 +39,37 @@ const githubEditUrl = `${CONFIG.GITHUB_EDIT_URL}/${currentFile}`;
     <title>
       {frontmatter.title} • {CONFIG.SITE.title}
     </title>
+    <script type="module" defer>
+      main();
+      function main() {
+        const section = document.querySelector("section.w-full");
+
+        const [headings1, headings2, headings3] = [
+          section.querySelectorAll("h1"),
+          section.querySelectorAll("h2"),
+          section.querySelectorAll("h3"),
+        ];
+
+        const headings = [...headings1, ...headings2, ...headings3];
+
+        headings.forEach((heading) => {
+          const link = document.createElement("a");
+          link.href = `#${heading.id}`;
+          link.className = "heading-link";
```

Comment:
Maybe change the stylings a bit here so they arent so " in your face".

Perhaps this is enough

```suggestion
          link.className = "hover:heading-link";
```

Reply:
It didn't

Reply:
[96cd482](https://github.com/t3-oss/create-t3-app/pull/639/commits/96cd4821c4da3a3243dcf998fa4a190557e8b0a6)